<ui version="4.0" >
 <class>dEditDialog</class>
 <widget class="Q3MainWindow" name="dEditDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>336</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget3" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Common</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="6" column="0" colspan="7" >
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="textLabel4" >
            <property name="text" >
             <string>Description:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Q3TextEdit" name="eDescription" />
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="textLabel1" >
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Module file</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="4" column="0" >
         <widget class="QLabel" name="textLabel1_3_2" >
          <property name="text" >
           <string>Form mode:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="6" >
         <widget class="QLineEdit" name="eModFile" />
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="textLabel1_2" >
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Name</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="textLabel1_3" >
          <property name="text" >
           <string>Use for:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="6" >
         <widget class="QLineEdit" name="eFormFile" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="3" >
         <widget class="QComboBox" name="cbFormMode" >
          <item>
           <property name="text" >
            <string>Dialog</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1" colspan="6" >
         <widget class="QLineEdit" name="eName" >
          <property name="readOnly" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="textLabel2" >
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Form file</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="3" column="1" >
         <widget class="QCheckBox" name="cbNew" >
          <property name="text" >
           <string>New</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="3" column="2" >
         <widget class="QCheckBox" name="cbView" >
          <property name="text" >
           <string>View</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="3" column="3" colspan="2" >
         <widget class="QCheckBox" name="cbEdit" >
          <property name="text" >
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="3" column="5" >
         <widget class="QCheckBox" name="cbSelect" >
          <property name="text" >
           <string>Selection</string>
          </property>
         </widget>
        </item>
        <item rowspan="2" row="3" column="6" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>30</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="4" colspan="3" >
         <widget class="QCheckBox" name="cbReadOnly" >
          <property name="text" >
           <string>Read only</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Design</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" colspan="3" >
         <widget class="Q3Frame" name="frame3" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>151</width>
            <height>171</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" >
         <widget class="QPushButton" name="pbDesigner" >
          <property name="text" >
           <string>Designer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QPushButton" name="bPreview" >
          <property name="text" >
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Module</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="ScriptEdit" native="1" name="eModule" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Aliases</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="Q3Table" name="tAliases" >
          <property name="numRows" >
           <number>0</number>
          </property>
          <property name="numCols" >
           <number>2</number>
          </property>
          <column>
           <property name="text" >
            <string>Lang</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction>rcIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>ScriptEdit</class>
   <extends>QWidget</extends>
   <header>scriptedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >mdtree.h</include>
  <include location="local" >formdesigner.h</include>
  <include location="local" >aliaseditor.h</include>
  <include location="local" >mainform.h</include>
  <include location="local" >scriptedit.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>bPreview</sender>
   <signal>clicked()</signal>
   <receiver>dEditDialog</receiver>
   <slot>formPreview()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDesigner</sender>
   <signal>clicked()</signal>
   <receiver>dEditDialog</receiver>
   <slot>EditForm()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
